File: Demo/MSVXcodeBuildNumberToCountGitCommits/MSVXcodeBuildNumberToCountGitCommits/ViewController.m
//
//  ViewController.m
//  MSVXcodeBuildNumberToCountGitCommits
//
//  https://github.com/sergemoskalenko/MSVXcodeBuildNumberToCountGitCommits
//  Created by Serge Moskalenko on 11/6/17.
//  Copyright © 2017 Serge Moskalenko. All rights reserved.
//

#import "ViewController.h"
#import <WebKit/WebKit.h>

@interface ViewController ()
@property (nonatomic) WKWebView *webView;
@end

@implementation ViewController

- (void)loadView {
    self.webView = [[WKWebView alloc] initWithFrame:CGRectZero configuration:[WKWebViewConfiguration new]];
    self.view = self.webView;
}

- (void)viewDidLoad {
    [super viewDidLoad];
}

- (void)viewWillAppear:(BOOL)animated {
    [super viewWillAppear:animated];
    [self.webView loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"http://camopu.rhorse.ru/MSVXcodeBuildNumberToCountGitCommits.html"]]];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}


@end

File: Demo-Swift/MSVXcodeBuildNumberToCountGitCommits-Swift/MSVXcodeBuildNumberToCountGitCommits-Swift/ViewController.swift
//
//  ViewController.swift
//  MSVXcodeBuildNumberToCountGitCommits-Swift
//
//  https://github.com/sergemoskalenko/MSVXcodeBuildNumberToCountGitCommits
//  Created by Serge Moskalenko on 11/6/17.
//  Copyright © 2017 Serge Moskalenko. All rights reserved.
//

import UIKit
import WebKit

class ViewController: UIViewController, WKUIDelegate {

    var webView: WKWebView!
    
    override func loadView() {
        webView = WKWebView(frame: .zero, configuration: WKWebViewConfiguration())
        webView.uiDelegate = self
        view = webView
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        webView.load(URLRequest(url: URL(string: "http://camopu.rhorse.ru/MSVXcodeBuildNumberToCountGitCommits.html")!))
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }


}

